### DISCUSSION TITLE
Can I filter dismiss_reviews to not dismiss on merge commits

### DISCUSSION THREAD

**Discussion by TheRealAmazonKendra** [Mergifyio/mergify#5037] [Q&A]
I'm trying to figure out if I can stop reviews from being dismissed when users just do a merge commit. I was thinking that maybe I could add a condition would not dismiss the review when the most recent commit has a title that begins with "Merge branch 'main' into " or something like that. It's a little blunt force and doesn't solve for 100% of our cases, but would cut down significantly on reviews getting dismissed just because someone does a merge.

**Comment by jd** [comment]
Hi @TheRealAmazonKendra 

There's no way, for now, to access the contents of the `commits` attribute to introspect the content of a commit message in the conditions. If that were possible, I think what you want to do would be possible.

Now, one trick is to update the PR using the [update](https://docs.mergify.com/commands/#update) command: that should not dismiss the reviews.

**Reply by TheRealAmazonKendra** [reply]
The problem is that our contributors don't necessarily use the update command. It dismisses when we have changes requested and resets several of our workflows. I'm looking for a way to limit that.
